sdk/core/azure-core/azure/core/configuration.py

```diff
@@ -74,6 +75,9 @@ def __init__(self, **kwargs):
         # Logger configuration
         self.logging_policy = None
 
+        # Http logger configuration
+        self.http_logging_policy = None
```

Comment:
Looks like we have some legacy issues. :(

It should be self.http_logging_policy = kwargs.get("http_logging_policy", None)?

Other properties have same issue.

Reply:
talked to @lmazuel . We're going to try to keep azure core behavior the same. I'll update the sphinx docstrings to be :ivar: instead of :keyword:

Reply:
Current design, this part was done by autorest, in the subclass of the object:
https://github.com/Azure/autorest.python/blob/autorestv3/test/vanilla/Expected/AcceptanceTests/BodyFile/bodyfile/_configuration.py

Don't ask me why it was done like that, we ported the logic from the `azure-core` branch :)

I guess one of the reason was also the default (not the same sync/async), so if the subclass host it it makes things easier.

Reply:
In fact I feel a little bit itchy on https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/configuration.py#L84. The ctor of configuration sets everything to None but consumes polling_interval.

Reply:
Btw, it does not block this PR. :)